spotbugs-exclude.xml

```diff
@@ -0,0 +1,9 @@
+<FindBugsFilter>
+    <Match>
+    <Bug pattern="URF_UNREAD_FIELD"/>
+    <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"/>
```

Comment:
Do we need to skip these? Doesn't these mean that there are unused fields in the code? If that's the case, shall we remove them, instead of adding a exclusions?

Reply:
There's a single field that causes this error in spotbugs. But we cannot remove that as it causes a test failure.